monai/transforms/utils.py

```diff
@@ -2051,5 +2058,46 @@ def has_status_keys(data: torch.Tensor, status_key: Any, default_message: str =
     return True, None
 
 
+def distance_transform_edt(
+    img: NdarrayOrTensor, sampling: None | float | list[float] = None, force_scipy: bool = False
+) -> NdarrayOrTensor:
+    """
+    Euclidean distance transform, either GPU based with CuPy / cuCIM
+    or CPU based with scipy.ndimage.
+    Choice depends on cuCIM being available or scipy can be forced with the ``force_scipy`` flag.
```

Comment:
@wyli @KumoLiu What is your opinion on how to choose between cpu and gpu? Should it just depend on Cucim being available or on whether the tensor is on GPU already?

Reply:
I didn't get the point here why do we need a `force_scipy` flag? Seems we can directly check the dependency and the device here.
https://github.com/Project-MONAI/MONAI/blob/e5f933781b1a5f4eb7d278e06389d531bff38dca/monai/transforms/utils.py#L1031C22-L1031C22

Reply:
I agree, since the callee that would have to pass the flag has control over the input tensor as well so they could just move it to CPU if scipy is desired.